tests/test-cases.js

```diff
@@ -1227,10 +1227,9 @@ var CUSTOM_TESTS = [
 				step: function(response, handle) {
 					updates.push(response.data[0]);
 					handle.abort();
-				},
-				complete: function() {
 					callback(updates);
```

Comment:
Complete is supposed to be called, but according to the documentation, "When streaming, parse results are not available in this callback." So I was using the callback inside to step function to verify the initial results, and then if the abort failed, the test case would fail with '(multiple results from test)'. I could make another test for verifying that complete is still called.